<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>724</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PSAT - Point Cloud Segmentation Annotation Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionSave"/>
    <addaction name="actionSetting"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionElevation"/>
    <addaction name="actionRgb"/>
    <addaction name="actionCategory"/>
    <addaction name="actionInstance"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPoint_size"/>
    <addaction name="actionPoint_size_2"/>
    <addaction name="actionBackground_color"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/language.png</normaloff>icons/language.png</iconset>
     </property>
     <addaction name="actionChinese"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="actionShortcut"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionGround_filter"/>
    <addaction name="actionClassify"/>
    <addaction name="separator"/>
    <addaction name="actionPick"/>
    <addaction name="actionCachePick"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpenFolder"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionTop_view"/>
   <addaction name="actionBottom_view"/>
   <addaction name="actionFront_view"/>
   <addaction name="actionBack_view"/>
   <addaction name="actionLeft_view"/>
   <addaction name="actionRight_view"/>
   <addaction name="separator"/>
   <addaction name="actionElevation"/>
   <addaction name="actionRgb"/>
   <addaction name="actionCategory"/>
   <addaction name="actionInstance"/>
   <addaction name="separator"/>
   <addaction name="actionGround_filter"/>
   <addaction name="actionClassify"/>
   <addaction name="actionPick"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>278</width>
     <height>645</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QWidget" name="info_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of points:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_num_point">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_size_x">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_size_y">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_size_z">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_offset_x">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_offset_y">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_offset_z">
             <property name="text">
              <string>None</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="label_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBox_showall">
             <property name="text">
              <string>Show all</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="label_listWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_files">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListWidget" name="listWidget_files"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/close.png</normaloff>:/icons/ui/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/close.png</normaloff>:/icons/ui/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/exit.png</normaloff>:/icons/ui/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPoint_size">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/point+.png</normaloff>:/icons/ui/icons/point+.png</iconset>
   </property>
   <property name="text">
    <string>Point size +</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="actionPoint_size_2">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/point-.png</normaloff>:/icons/ui/icons/point-.png</iconset>
   </property>
   <property name="text">
    <string>Point size -</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionTop_view">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/topview.png</normaloff>:/icons/ui/icons/topview.png</iconset>
   </property>
   <property name="text">
    <string>Top view</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionBottom_view">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/bottomview.png</normaloff>:/icons/ui/icons/bottomview.png</iconset>
   </property>
   <property name="text">
    <string>Bottom view</string>
   </property>
  </action>
  <action name="actionFront_view">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/frontview.png</normaloff>:/icons/ui/icons/frontview.png</iconset>
   </property>
   <property name="text">
    <string>Front view</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionBack_view">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/backview.png</normaloff>:/icons/ui/icons/backview.png</iconset>
   </property>
   <property name="text">
    <string>Back view</string>
   </property>
  </action>
  <action name="actionLeft_view">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/leftview.png</normaloff>:/icons/ui/icons/leftview.png</iconset>
   </property>
   <property name="text">
    <string>Left view</string>
   </property>
  </action>
  <action name="actionRight_view">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/rightview.png</normaloff>:/icons/ui/icons/rightview.png</iconset>
   </property>
   <property name="text">
    <string>Right view</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionBackground_color">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/背景颜色_background-color.svg</normaloff>:/icons/ui/icons/背景颜色_background-color.svg</iconset>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
  </action>
  <action name="actionElevation">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/等高线_21_21.png</normaloff>:/icons/ui/icons/等高线_21_21.png</iconset>
   </property>
   <property name="text">
    <string>Elevation</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionRgb">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/RGB.png</normaloff>:/icons/ui/icons/RGB.png</iconset>
   </property>
   <property name="text">
    <string>Rgb</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionCategory">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/semantic.png</normaloff>:/icons/ui/icons/semantic.png</iconset>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
   <property name="toolTip">
    <string>Category</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionInstance">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/instance.png</normaloff>:/icons/ui/icons/instance.png</iconset>
   </property>
   <property name="text">
    <string>Instance</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionGround_filter">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/ground21.png</normaloff>:/icons/ui/icons/ground21.png</iconset>
   </property>
   <property name="text">
    <string>Ground filter</string>
   </property>
  </action>
  <action name="actionClassify">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/classify-21.png</normaloff>:/icons/ui/icons/classify-21.png</iconset>
   </property>
   <property name="text">
    <string>Classify</string>
   </property>
  </action>
  <action name="actionPick">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/polygon.png</normaloff>:/icons/ui/icons/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Pick</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/setting.png</normaloff>:/icons/ui/icons/setting.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionCachePick">
   <property name="text">
    <string>Cache pick</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/save.png</normaloff>:/icons/ui/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionShortcut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/keyboard.png</normaloff>:/icons/ui/icons/keyboard.png</iconset>
   </property>
   <property name="text">
    <string>Shortcut</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/about.png</normaloff>:/icons/ui/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/ui/icons/open.png</normaloff>:/icons/ui/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
